src/plots/geo/layout/index.js

```diff
@@ -1,18 +0,0 @@
-/**
-* Copyright 2012-2016, Plotly, Inc.
-* All rights reserved.
-*
-* This source code is licensed under the MIT license found in the
-* LICENSE file in the root directory of this source tree.
-*/
-
-
-'use strict';
-
-var Plotly = require('../../../plotly');
-var attributes = require('./attributes');
-
-Plotly.Plots.registerSubplot('geo', 'geo', 'geo', attributes);
-
-exports.layoutAttributes = require('./layout_attributes');
-exports.supplyLayoutDefaults = require('./defaults');
```

Comment:
no need for the `GeoLayout` and `Gl3dLayout` modules anymore. Their content is merged into `plots/geo/index.js` and `plots/gl3d/index.js` respectively.